support/controlplane-component/status.go

```diff
@@ -89,6 +89,14 @@ func (c *controlPlaneWorkload) reconcileComponentStatus(cpContext ControlPlaneCo
 		return err
 	}
 
+	if c.serviceAccountKubeConfigOpts != nil {
```

Comment:
can we also check `hyperv1.DisablePKIReconciliationAnnotation` annotation here.

Reply:
done